airflow/providers/amazon/aws/operators/cloud_formation.py

```diff
@@ -44,17 +44,17 @@ class CloudFormationCreateStackOperator(BaseOperator):
     template_ext: Sequence[str] = ()
     ui_color = '#6b9659'
 
-    def __init__(self, *, stack_name: str, params: dict, aws_conn_id: str = 'aws_default', **kwargs):
+    def __init__(self, *, stack_name: str, parameters: dict, aws_conn_id: str = 'aws_default', **kwargs):
```

Comment:
This makes it a breaking change to the operator.

Reply:
(I'm okay with that, just FYI when releasing. Not sure if we need to note that here in this PR etc/)

Reply:
Yes. I realize that. I made a mental note about it but you are right I should add it to CHANGELOG now so that I won't forget

Reply:
Separated it out and added breaking notes in #20989